cmd/client.go

```diff
@@ -465,7 +466,30 @@ var clientTlsCmd = &cobra.Command{
 			log.Fatalf("failed to connect: %v", err.Error())
 		}
 
-		tcp_con_handle(conn)
+		if listener > 0 {
+			l, err := net.Listen("tcp", fmt.Sprintf("localhost:%d", listener))
+			if err != nil {
+				log.Fatalln("Error: Unable to start local TLS listener.")
+			}
+
+			var wg sync.WaitGroup
+			wg.Add(1)
+
+			go func() {
+				defer wg.Done()
+				log.Printf("Waiting for connection...")
+				lcon, err := l.Accept()
+				if err != nil {
+					log.Printf("Listener: Accept Error: %s\n", err)
```

Comment:
Should we change it to `Fatalf`? the line below will try to close the connection when the function returns `lcon` might be nil at that point.

Reply:
You're right, I changed it to fatalf.